utils/merge.py

```
@@ -160,37 +247,37 @@ def make_author_model(book, authors: list[dict[str, str]]):
     # Author DB entry
     # Create new entry for each author if there's more than one
     for author in authors:
-        author_name_full = author['name']
+        author_name_full = author.get("name", "")
         author_name_split = author_name_full.split()
+
+        # Skip authors with empty names
+        if not author_name_split:
+            logger.warning(f"Skipping author with empty name")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)